src/ucp/core/ucp_ep.h

```diff
@@ -166,8 +166,11 @@ enum {
     UCP_EP_INIT_CREATE_AM_LANE_ONLY    = UCS_BIT(8),  /**< Endpoint requires an AM lane only */
     UCP_EP_INIT_KA_FROM_EXIST_LANES    = UCS_BIT(9),  /**< Use only existing lanes to create
                                                            keepalive lane */
-    UCP_EP_INIT_ALLOW_AM_AUX_TL        = UCS_BIT(10)  /**< Endpoint allows selecting of auxiliary
+    UCP_EP_INIT_ALLOW_AM_AUX_TL        = UCS_BIT(10), /**< Endpoint allows selecting of auxiliary
                                                            transports for AM lane */
+    UCP_EP_INIT_FLAG_PRIORITIZED       = UCS_BIT(11)  /**< Endpoint is prioritized due to a
```

Comment:
whre is this flag set?

Reply:
on a separate PR

Reply:
can share the link?

Reply:
it's not yet ready, will share when it's opened

Reply:
without usage it's not clear why do we need another "est_num_eps" parameter. Also could pls point me to
> Set latency multiplier to zero if EP_PROMOTED is set (because num_eps should be ignored).

Reply:
the extra est_num_eps is for promoted EPs latency calculation (because we need to get better latency so cannot use the regular est_num_eps in case it's big).
The comment abount the multiplier is not relevant anymore, updated the description.

Reply:
lets move this flag to next PR, the main idea in this PR is to add new parameter which affects latency calculation, the rest is uncler without usage